packages/sui-widget-embedder/development/replace-middleware.js

```diff
@@ -0,0 +1,55 @@
+/* eslint no-nested-ternary:0 */
+const Gunzip = require('zlib').Gunzip
+
+const CONTENT_TYPE_HTML = /text\/html/
+const GZIP_ENCODING = /gzip/i
+
+/**
+ * Where the magic happen!
+ * */
+const replace = (buff) => {
+  const tag = 'body'
+  const snippet = `
+    <script async src="/bundle.js"></script>
+    </${tag}>
+  `
+  return buff.toString('utf8').replace(`</${tag}>`, `${snippet}`)
```

Comment:
Such a clever way to replace the desired tag with your widget or whatever snippet